plugins/modules/zos_backup_restore.py

```diff
@@ -186,15 +186,15 @@
   hlq:
     description:
       - Specifies the new HLQ to use for the data sets being restored.
-      - Defaults to running user's username.
+      - Defaults to running original HLQ.
     type: str
     required: false
   tmp_hlq:
     description:
       - Override the default high level qualifier (HLQ) for temporary and backup
         data sets.
-      - The default HLQ is the Ansible user that executes the module and if
-        that is not available, then the value of C(TMPHLQ) is used.
+      - The default is the original HLQ fo the data sets if that is not available,
+        then the value of C(TMPHLQ) is used.
```

Comment:
we were talking about the default behavior of the tmp hlq offline, just wanted to confirm that this statement is accurate. 

also `fo` should be `of`.